weaver/core/network/fabric-interop-cc/contracts/interop/VERSION

```diff
@@ -1 +1 @@
-2.0.0
\ No newline at end of file
+2.1.0
```

Comment:
Hi @petermetz Can you run `go-gen-checksum` script? If already done please try again, check if there was some error? The go.mod and go.sum isn't updated anywhere

Reply:
@sandeepnRES I re-ran the script but it doesn't produce any diff unfortunately. Please see the output:

```sh
$ ./tools/go-gen-checksum.sh 2.1.0 .
Installing go-checksum...
Installed.
REPO: github.com/hyperledger/cacti
VERSION: 2.1.0
############# START weaver/core/network/fabric-interop-cc/libs/utils ################
############# END weaver/core/network/fabric-interop-cc/libs/utils ################\n
############# START weaver/core/network/fabric-interop-cc/libs/assetexchange ################
############# END weaver/core/network/fabric-interop-cc/libs/assetexchange ################\n
############# START weaver/core/network/fabric-interop-cc/interfaces/asset-mgmt ################
############# END weaver/core/network/fabric-interop-cc/interfaces/asset-mgmt ################\n
############# START weaver/core/network/fabric-interop-cc/contracts/interop ################
############# END weaver/core/network/fabric-interop-cc/contracts/interop ################\n
############# START weaver/sdks/fabric/go-sdk ################
############# END weaver/sdks/fabric/go-sdk ################\n
############# START weaver/samples/fabric/go-cli ################
############# END weaver/samples/fabric/go-cli ################\n
############# START weaver/samples/fabric/simpleasset ################
############# END weaver/samples/fabric/simpleasset ################\n
############# START weaver/samples/fabric/satpsimpleasset ################
############# END weaver/samples/fabric/satpsimpleasset ################\n
############# START weaver/samples/fabric/simpleassetandinterop ################
############# END weaver/samples/fabric/simpleassetandinterop ################\n
############# START weaver/samples/fabric/simpleassettransfer ################
############# END weaver/samples/fabric/simpleassettransfer ################\n
############# START weaver/samples/fabric/simplestatewithacl ################
############# END weaver/samples/fabric/simplestatewithacl ################\n
############# START weaver/samples/fabric/simplestate ################
############# END weaver/samples/fabric/simplestate ################\n
$ 
$ 
$ git status
On branch release-v2.1.0
Your branch is up to date with 'upstream/release-v2.1.0'.

nothing to commit, working tree clean
$ 
```

I'm going to try and investigate it more now. If you have any ideas please let me know as well!

Reply:
@sandeepnRES FYI I'm committing the following fix:
![image](https://github.com/user-attachments/assets/1e91ac9d-8ddc-450f-9029-24746aada95e)